plugins/woocommerce-admin/client/header/index.js

```diff
@@ -31,7 +31,7 @@ export const PAGE_TITLE_FILTER = 'woocommerce_admin_header_page_title';
 
 export const getPageTitle = ( sections ) => {
 	let pageTitle;
-	const pagesWithTabs = [ 'Settings', 'Reports', 'Status' ];
+	const pagesWithTabs = [ 'Settings', 'Reports (deprecated)', 'Status' ];
```

Comment:
This is non-localized hardcoded value.
The fix for that is in https://github.com/woocommerce/woocommerce/pull/51606